nais.yaml

```diff
@@ -22,7 +22,7 @@ spec:
     enabled: {{prometheusEnabled}}
     path: /actuator/metrics
   replicas:
-    min: 1
+    min: 2
```

Comment:
Trenger vi flere replicaer for å validere filnavn? Eller er det en annen grunn? :)

Reply:
Øker også min-replicas til 2 slik at det alltid er minst 2 podder som kjører. (da vil det ikke bli nedetid om en brått skulle gå ned.